.github/workflows/tensorflow_setup.yml

```diff
@@ -0,0 +1,136 @@
+name: Test for DeepChem Tensorflow
+on:
+  push: # ci work when pushing master branch
+    branches:
+      - master
+  pull_request: # ci work when creating a PR to master branch
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
```

Comment:
Maybe we should just go ahead and enable 3.8 here as well given we're using it for Jax